module5/lessons/Week1/Logging.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: page
+title: Logging
+---
+
+## Learning Goals
+* Understand the benefits of logging
+* Be able to implement logging to the console and a file in a .NET application
+* Practice the skill of self-directed learning and building
+
+## Collecting Questions
+
+In preparation for today's lesson, you were asked to imagine that you are assigned the task "Add logging to our MVC Movies application.". Then you were asked to brainstorm questions you would want to figure out answers to before you even think about how to write the code for this task.
+
+<section class="call-to-action" markdown="1">
+In your group of 3-4, share the questions each of you came up with. Then narrow down to the three questions that seem most important. <br> <br> When we come back together your instructor will call on groups and compile a list of top questions in Slack, each as a separate post.
+</section>
+
+<aside class="instructor-notes" markdown="1">
+    <p><strong>Instructor Note</strong><br>
+    Other possible questions could be: Why do we use logging? What should I log? Where do we want these logs go? Is logging already a part of .NET or is it some other tool I'll need to learn? <br>
+    We want to direct students to write to a file and the console.</p>
+</aside>
+
+❓If you were really on a dev team, how might you go about figuring out the answers to these questions?

```

Comment:
I'm going to move this into a full call to action section, because I do want to make sure students are brainstorming strategies.

I also like the idea of mentioning writing this down in some sort of "Tools for my first Job"